src/main/proto/ga4gh/coverage_service.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package ga4gh;
+
+// The coverage service returns binned counts for on requested containers.
+// It is designed to provide intermediate scale views of the data for genome
+// browsers.
+service CoverageService {
+  // Gets a list of `Dataset` matching the search criteria.
+  //
+  // `POST /datasets/search` must accept a JSON version of
+  // `SearchDatasetsRequest` as the post body and will return a JSON
+  // version of `SearchDatasetsResponse`.
+  rpc Coverage(CoverageRequest)
+    returns (CoverageResponse) {
+      option (google.api.http) = {
+        post: "/v0.6.0a8/coverage"
+        body: "*"
+      };
+    };
+}
+
+// A Bin message provides the 
+message Bin {
+  // The number of features in this bin.
```

Comment:
That might be a good idea, especially if the bin width is less than the requested one. For example, I expect a client to know what the bin width is based on their request. If the width is different than requested it can be added into the response (in the case where a server precalculates specific widths).

However, that misses the case when the region is 5 wide and I select a binning of 2. That means the last bin is 1-wide, in which case the client would probably like to know that. Putting a width in every bin seems redundant though, maybe clients check for the presence of a width in a bin as an optional field?